dask_cuda/benchmarks/utils.py

```diff
@@ -176,8 +183,11 @@ def get_scheduler_workers(dask_scheduler=None):
 
 
 def setup_memory_pool(pool_size=None, disable_pool=False):
-    import rmm
-    import cupy
+    try:
+        import rmm
+        import cupy
```

Comment:
It doesn't make sense to have `import cupy` inside the `try` block for this test, for any GPU test CuPy is required.

Reply:
That's true, however, when loading utils in `local_cudf_merge.py` this will fail unless cupy is installed or in the try block